portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/usecase/BlogsUsecase.testcase

```diff
@@ -333,6 +333,35 @@
 		</execute>
 	</command>
 
+	<command name="ChangeBlogsRatingType" priority="4">
+		<execute macro="Blogs#addEntry">
+			<var name="entryContent" value="Blogs Entry Content" />
+			<var name="entryTitle" value="Blogs Entry Title" />
+		</execute>
+
+		<execute macro="ProductMenu#gotoPortlet">
+			<var name="category" value="Configuration" />
+			<var name="panel" value="Control Panel" />
+			<var name="portlet" value="Instance Settings" />
+		</execute>
+
+		<execute macro="Navigator#gotoNavTab">
+			<var name="navTab" value="Social" />
+		</execute>
+
+		<execute macro="PortalSettings#configureBlogsRatingType">
+			<var name="ratingType" value="Stars" />
+		</execute>
+
+		<execute macro="Navigator#openURL" />
+
+		<execute macro="Navigator#gotoPage">
+			<var name="pageName" value="Blogs Page" />
+		</execute>
+
+		<execute function="AssertTextEquals" locator1="Ratings#AVERAGE_LABEL" value1="Average (0 Votes)" />
```

Comment:
Hey @andrewlam78, one more thing - can you also assert that the stars (not just the text) appear? Thanks!